test/e2e_node/cpu_manager_metrics_test.go

```
@@ -274,6 +290,88 @@ var _ = SIGDescribe("CPU Manager Metrics", framework.WithSerial(), feature.CPUMa
 			ginkgo.By("Ensuring the metrics match the expectations a few more times")
 			gomega.Consistently(ctx, getKubeletMetrics, 30*time.Second, 10*time.Second).Should(matchResourceMetricsIdle)
 		})
+
+		ginkgo.It("should update alignment counters when pod successfully run taking less than uncore cache group", func(ctx context.Context) {
+			if !hasSplitUncore {
+				e2eskipper.Skip("Skipping CPU Manager uncore alignment test - not split Uncore detected")
+			}
+			if smtLevel >= uncoreGroupSize { // how come?!
```

Can you elaborate on this?

---

sure, will fix the comment. I don't ever expect HW (now or ever) which have thread siblings span across uncore groups, if nothing else because this doesn't make sense from the very definition of uncore cache. So this has to be a configuration or detection error. Thinking about it, this should case the test to fail loudly. I'll change it. Thanks for noticing!